src/Lhm/Chunker.php

```diff
@@ -86,13 +90,25 @@ public function __construct(AdapterInterface $adapter, \Phinx\Db\Table $origin,
         $this->sqlHelper = $sqlHelper ?: new SqlHelper($this->adapter);
 
         $this->options = $options + [
-                'stride' => 2000,
-                'strideMax' => 20000,
-                'targetQuerySeconds' => 0.75
-            ];
+            'stride' => 2000,
+            'strideMax' => 20000,
+            'targetQuerySeconds' => 0.75,
+        ];
 
         $this->primaryKey = $this->adapter->quoteColumnName($this->sqlHelper->extractPrimaryKey($this->origin));
 
+        $adapterFactory = AdapterFactory::instance();
+        $res = $this->adapter->query('SELECT HOST FROM INFORMATION_SCHEMA.PROCESSLIST WHERE COMMAND = \'Binlog Dump\'')->fetchAll();
+        foreach ($res as $row) {
```

Comment:
do I need to pull $row['HOST'] instead?? I saw some snippets that make it seem that way and others where it doesn't

Reply:
They both work (both is the default)